### PR TITLE
Feature/add examples

### PR DESCRIPTION
**Pull request by alexandra-hanson**
This adds examples from Eric's repository here: https://github.com/ericvanwyk/ldta-2011. I believe these were the same examples used for testing from the tool challenge (they look like they've just been copied over). I also wrote a simple script to run the given examples in a directory; you could take a look at `examples/positive/L1/run` and the rest of the scripts are all the same.

I also added a note to the README about where these examples came from and how to run them.

### CODE REVIEW

**Comment by krame505** [comment]
Looks good!  Could you add these to the Jenkinsfile so that they get integration tested, too?

**Comment by alexandra-hanson** [comment]
Okay, although it is still failing, I believe the Jenkinsfile is working as I expect it to now... I'm still new to Jenkins though so it might be good if someone could let me know if I've done things properly.

From what I can tell from the console output, at least one of my scripts is returning with exit code 1, which is what I expected. There is a bug where record fields produce undeclared value errors after they've been declared (see issue #4) so examples/positive/L4/record.ob is failing. Additionally, I've noticed that examples/negative/L3/8_non_local_var_proc.ob also seems to be failing since it does not produce any errors... I'll need to take a closer look at that.

**Comment by alexandra-hanson** [comment]
I removed the two tests that were failing but still left those examples within the examples directory. Let me know if that is an issue at all, as well as if the Jenkinsfile looks okay.

File: examples/negative/name_errors/L3/run [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Specify which artifact jar you would like to use, e.g. oberon0_a5.jar
+
+ObJar="oberon0_a5.jar"
+
+JAR=${PathToJar}${ObJar}
+
+if [ "$1" == "test" ];
+then
+  # Build all examples and test that a .c file is NOT generated, as expected
+  # TO DO: add back in "8_non_local_var_proc" to examples array below once
```
Comment (krame505) [inline examples/negative/name_errors/L3/run:19]: Maybe add a link to the github issue here

**Review by krame505 — APPROVED** [review]

**Pull request merged**